01-messaging.md

```diff
@@ -918,7 +918,7 @@ message. The base `init` message (without extensions) for these examples is
 The following `init` messages are valid:
 
 - `0x001000000000`: no extension provided
-- `0x00100000000001012a030104`: the extension contains two _odd_ TLV records (with types `0x01` and `0x03`)
+- `0x00100000000009012a0b0104`: the extension contains two unknown _odd_ TLV records (with types `0x09` and `0x0b`)
```

Comment:
Could you use even higher values, so that we're pretty sure we won't have to update it again in 6 months? For example `201` and `203`?

Could you also update the tests in the section below:

- `0x00100000000002012a` should use tlv type `202` instead of `2` otherwise it's likely it will be wrong soon
- `0x001000000000010101010102` should probably use tlv type `201` instead of `1`, right now it's invalid mostly because tlv type 1 should be 32 bytes, not necessarily because the type is duplicated

Thanks!

Reply:
gave it a shot, agreed